# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Parent  cbeee3a748c21a77e3f2b1e329b2fe40b9b953d8
Bug 2012358 follow-up: Revert 'Display the full folder path for rule action targets in the filter editor'.

Displaying the path ...
- makes it inconsistent with other places where the folder name is displayed
- causes the tooltip to be identical with the target folder display
- doesn't work for RTL locales (which aren't supported in Betterbird).

diff --git a/mailnews/search/content/searchWidgets.js b/mailnews/search/content/searchWidgets.js
--- a/mailnews/search/content/searchWidgets.js
+++ b/mailnews/search/content/searchWidgets.js
@@ -139,9 +139,9 @@
           `
           <menulist class="ruleactionitem
                     folderMenuItem"
                     flex="1"
-                    displayformat="path">
+                    displayformat="verbose">
             <menupopup is="folder-menupopup"
                        mode="filing"
                        class="menulist-menupopup"
                        showRecent="true"
